modules/UI/authentication/AuthHandler.js

```diff
@@ -80,8 +82,11 @@ function initJWTTokenListener(room) {
             return;
         }
         if (event.data && event.data.jwtToken) {
-            config.token = event.data.jwtToken;
-            logger.info("Received JWT token:", config.token);
+            const token = event.data.jwtToken;
+
+            APP.store.dispatch(setTokenData({ token }));
```

Comment:
We need to parse the token, we can't store it just like that, it's not an instance of TokenData.